src/Bridges/ApplicationLatte/TemplateFactory.php

```diff
@@ -112,4 +115,12 @@ public function createTemplate(UI\Control $control = NULL)
 		return $template;
 	}
 
+	public function setTemplateClass($templateClass)
+	{
+		if (!class_exists($templateClass) || !is_a($templateClass, UI\ITemplate::class, TRUE)) {
```

Comment:
It has `@return Template`, it assumes `__construct(Latte\Engine)` and `__set` for variables (or fields).